.github/ISSUE_TEMPLATE/feature.yaml

```diff
@@ -0,0 +1,59 @@
+name:  🔧 feature request
+title: "[Feature]: "
+description: "request the team work on new feature"
+labels: ["enhancement", "feature", "review"]
```

Comment:
What about just feature and triage? Is "review" functioning like "triage" is on bugs?

Reply:
ya, that makes sense, I actually did want the triage tag. @arbitrarytech I'm going to add a triage tag to our "defined set" if no concerns.